doc/modules/cassandra/pages/developing/cql/constraints.adoc

```diff
@@ -92,6 +92,31 @@ Finally, the constraint can be removed:
 ALTER TABLE keyspace.table ALTER name DROP CHECK;
 ----
 
+=== OCTET_LENGTH CONSTRAINT
+
+Defines a condition that checks the size in bytes of text or binary type.
+
+For example, we can create a constraint that checks that name can't be bigger than 256 characters:
+
+----
+CREATE TABLE keyspace.table (
+	name text CHECK OCTET_LENGTH(name) < 256
+	...,
+);
+----
+
+Altering that constraint can be done with:
+
+----
+ALTER TABLE keyspace.table ALTER name OCTET_LENGTH(name) < 512;
+----
+
+Finally, the constraint can be removed:
+
+----
+ALTER TABLE keyspace.table ALTER name DROP CHECK;
```

Comment:
I would not do this. This is obvious already. I would put there examples of the usage instead. Happy path and how failure would look like.